python/pyproject.toml

```diff
@@ -25,7 +22,7 @@ python-dotenv = "^1.0.1"
 magika = "magika.cli.magika:main"
 
 [tool.poetry.group.dev.dependencies]
-pytest = "^7.4.0"
+pytest = ">=7.4,<9.0"
```

Comment:
what is the difference between ^7.4 and >=7.4?

Reply:
This actually needed to be fixed, pytest is now v8.0.1, while doing this also fixed `ipython` which versions change based on your Python version.